target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -335,11 +335,19 @@ wndr3700v5)
 wt3020)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "0"
 	;;
-y1|\
+y1)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
+	set_usb_led "$board:blue:usb"
+	set_wifi_led "$board:blue:wifi" "wlan1"
```

Comment:
use ucidef_set_led_netdev for the 2.4 GHZ as well. Don't different functions for 2.4 and 5 GHz wifi. It is confusing.